ceph/exporter.go

```diff
@@ -86,6 +91,42 @@ func (exporter *Exporter) cephVersionCmd() []byte {
 	return cmd
 }
 
+func (exporter *Exporter) cephVersionsCmd() []byte {
+	cmd, err := json.Marshal(map[string]interface{}{
+		"prefix": "versions",
+	})
+	if err != nil {
+		exporter.Logger.WithError(err).Panic("failed to marshal ceph versions command")
+	}
+
+	return cmd
+}
```

Comment:
This is also implemented in `monitor.go`, including parsing. Maybe factor it out into a common place? Ideally we would run each command exactly once but I think that's a bit more work, so it's fine if it runs twice.

Reply:
Okay, I think I got this a little better. i moved the cmd call to exporter and made it public, then added the parse function there as well.

Reply:
:+1: If you wanted to take it one step further you could roll the run-command-and-parse into one function, but that's up to you.